tests/testthat/test-outfiles.R

```diff
@@ -26,6 +27,7 @@ test_that("expected outfiles are written and messaged, venue = 'gh'", {
 test_that("expected outfiles are written and messaged, venue = 'R'", {
   skip_on_cran()
   local_temp_wd()
+  local_reprex_quiet(FALSE)
 
   msg <- capture_messages(
```

Comment:
Could you explain a bit more what you're trying to test here? I suspect there's some combination of `expect_snapshot()` + `expect_message()` that will cover what you want.

Reply:
(Obviously also fine to keep using `capture_messages()`)

Reply:
I re-applied myself to this and could convert some of these to snapshots tests. Except for the problem that some of the messaging contains temporary file paths. I'm going to stick with my specific expectations about the message content for now.

```
Failure (test-outfiles.R:14:3): expected outfiles are written and messaged, venue = 'gh'
Snapshot of code has changed:
old[9:12] vs new[9:12]
  "  Rendering reprex..."
  "Message <reprex_message>"
  "  Writing reprex file:"
- "  * /private/var/folders/pv/l2mbh7j90y5_vslh_1wyqxj00000gn/T/RtmpVagDDM/reprextests89117e64c489/foo_reprex.md"
+ "  * /private/var/folders/pv/l2mbh7j90y5_vslh_1wyqxj00000gn/T/Rtmpex7oWJ/reprextests8cfd3b31b0c8/foo_reprex.md"

Run `snapshot_accept('outfiles')` if this is a deliberate change
```

Reply:
Yeah I think in this scenario I’d either fix the path or mock whatever prints the path. But I don’t think you need to change this code - I just want to understand how we’d solve with today’s tools.